docs/en/pact-functions.md

```diff
@@ -359,7 +359,7 @@ pact> (int-to-str 64 43981)
 *charset*&nbsp;`integer` *input*&nbsp;`string` *&rarr;*&nbsp;`bool`
 
 
-Check that a string INPUT conforms to the a supported character set CHARSET.       Character sets currently supported are: 'CHARSET_LATIN1' (ISO-8859-1), and         'CHARSET_ASCII' (ASCII). Support for sets up through ISO 8859-5 supplement will be added in the future.
+Check that a string INPUT conforms to the supported character set CHARSET.       Character sets currently supported are: 'CHARSET_LATIN1' (ISO-8859-1), and         'CHARSET_ASCII' (ASCII). Support for sets up through ISO 8859-5 supplement will be added in the future.
```

Comment:
These need to be updated in the corresponding native function in `Pact.Natives.hs`. The documentation in `pact-functions.md` is derived from the haskell code.

Reply:
Sure! just changed this file too

Reply:
We cannot change docs in natives at this time as there is the possibility that their docs are outputted on-chain. We need a back-compat solution before we can update these docs.

Reply:
Also echoing @emilypi -- please do not directly edit `pact-functions.md` or `pact-functions.rst` as these are auto-generated, and since we have the issue above with docs, cannot fix those docs now. The edits to pact-reference.md are welcome though!